### PR TITLE
Adds a new map

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

image

A map that is almost done. ETA less than a week.

## Changelog

:cl:
add: Adds a new map called Pubberstation
/:cl:

### CODE REVIEW

**Comment by Oscari-pog** [comment]
Gonna need a trailer video and a pre-order option for this one

**Comment by BurgerLUA** [comment]
image

Map is pretty much (almost) done. Just needs request consoles added and it'd be good to go for testing.

**Comment by Detective-Google** [comment]
PUBBY MY SWEETUMS YOURE BACK

please tell me you kept the shuttle/escape pod combo thing it was the best part of this map (that and the train shuttle)

**Comment by BurgerLUA** [comment]
> PUBBY MY SWEETUMS YOURE BACK
> 
> please tell me you kept the shuttle/escape pod combo thing it was the best part of this map (that and the train shuttle)

I hate to inform you but while this map was meant to be a recreation of Pubbystation, it ended up straying really far from that.

**Comment by BurgerLUA** [comment]
The map is in a testable state. SM setup works, disposals work, no mapping errors, no secret pulse rifle spawns.

Just keep in mind that it is entirely possible that I forgot to add or do something, like some door with access or some extra detail in an area.

**Comment by Detective-Google** [comment]
> > PUBBY MY SWEETUMS YOURE BACK
> > please tell me you kept the shuttle/escape pod combo thing it was the best part of this map (that and the train shuttle)
> 
> I hate to inform you but while this map was meant to be a recreation of Pubbystation, it ended up straying really far from that.

but it does have the big pod right anon...

**Comment by BurgerLUA** [comment]
It is testable for real:tm: now.

SM works.
Evac works.
ERT Shuttle works.
Mining shuttle works.
Cargo shuttle works.
Whiteship dock works.
No compile errors.
No CI errors (for the map at least).
Works on my machine:tm:
Dorms work.

**Comment by stoneman2** [comment]
Hello, medical player here. I've come to ask a few things, alongside concerns with other people, to see what we could do to make this work even better, feedback starting:
Overall: Love the layout. Great, everything's pretty neatly placed around in other parts of the station!

But I gotta really start nitpicking! Just so QOL of playing here is even better than it is.
-Medbay Storage: 
I feel like the tables need secure windows to stop people from just yoinking the treatment kits like other maps. 
Also, damn.. five doctors locker.. 

The disposals in the corner of the room is unreachable, so if cargo sends crates through disposals, it is literally stuck between a table and the suit storage unit. I know you can open it, but it is a little awkward!

Can we remove one of the doctor's lockers and replace it with a medical clothes vendor. We lack that.

You can't reach the cell charger on the top left corner either.

The sheer amount of defibs. Included in the doctor's locker, that's 8 defibs... damn.

-Treatment center: 
Great we got crew monitoring console at round start.

I don't like that we have so many beds in the treatment center. From my experience, most don't even use these and removes them for more space for stasis beds.

On the topic of stasis beds, please.. pretty please, can we have two (or one) more, replacing the beds?

We don't have any defib mounts. Pretty crucial!

No stethoscope. No empty syringes. All maps at least got them lying about.

-Surgery Rooms:
This is a big one.. we don't have surgical tools at round start. Every map has at least two bags of surgical tools in surgery rooms. This place don't have any.

-Chemistry:
Only one chem dispenser! Every map has 2! This won't do! Noooo!!! Please one more!

We also lack a screwdriver, wires, grenade parts on the tables.

-Virology:
No monkeys?

We also don't have syringe and beaker boxes! Every virology has it in every other map.

No showers?

-Others:
I'm unsure about the gameplay, but only one anesthetic mix for cryo?
We don't have cryoxadone! Maps always spawn with four!

Unsure about genecist being in medbay either! Never had them interact much with medical in my days, but I am indifferent.
The three differently colored beds in the medbay halls, I don't know what's that about. Can we replace them with more 
wheelchairs and roller beds? Two of each maybe? Instead of one of each and beds.

We're lacking space cleaner sprays.. can we have one or two in each surgery room? Or just one somewhere. All maps got it.

No modsuit chargers! Every map got one! 

We don't have a medical break room!! NOOO!!!

CMO don't have a health hud! If CMO is allergic to implants, they can't take it. Believe other maps gives the stuff in the locker.

CMO don't have a screwdriver in the locker.


That is all that I've noticed for now. I'll add more if I see anything else.

**Comment by RemervKeberb** [comment]
Looks great! Only a few things stand out.

Maints feels like it's all lanky corridors. Have just one or two extra proper room areas would make tiding and maints dens a lot more appealing.

Toolstorage has no GPS, might be intentional but I'll bring it up since most maps have one or two there.

There's no auxiliary EVA airlocks around half the map. The only real way to get out of the station is via the top right by arrivals and the other standard methods such as the cargo dock and engineering. An extra EVA door by the top left near the pool or in the maints besides it would go a long way.

The blueshield having their own office is good, however as an assistant I flew out into EVA. went to the external docking port by his office. Had access to that so now I was in a command area once I entered. But instead of just being trapped with no choice but to go back out. I could walk into the blueshields office. Their door doesn't have access restrictions properly in place.
Probably wanna fix that along with the docking port itself but I'm unsure what's actually meant to be there.

There's nothing fucky in terms of the z-levels. Space exploration seems fine.

All in all, great stuff. Those are the few things you might wanna look over though.

**Comment by CenkTheRaptor** [comment]
i like this map for the most part. but things like the captain's office/quarters getting plasma glass seems a bit much, along with the general reinforcement/strategic layout of major antagonist objectives.

for example, the captain's quarters really only has one viable entrance path: maintenance, as anywhere else would be too public, and even then it is incredibly risky due to there only being a few walls between it and the bridge, it would be easily heard. overall i do not like how key areas like that are given extra reinforcement specifically to make it much harder for antagonists to reach, without much more preparation and luck.

**Comment by BurgerLUA** [comment]
99% of above have been fixed
1% is what I'm working on right now

**Comment by YakumoChen** [comment]
det office has a bunch of lawyer file cabinets instead of empty ones
the dept-maint doors need labeled, otherwise its impossible to know where any door leads to in maint

lack of r-walls in general in places that should really have r-walls (the captain office is the most glaring example)

**Comment by YakumoChen** [comment]
Security has no red clothes vendor?

**Comment by YakumoChen** [comment]
Law offices need shutters

**Comment by YakumoChen** [comment]
Very noticeable lack of xeno spawn locations, and only one of them is near an air vent (which is semi-vital to xeno larva living for long)

**Comment by BurgerLUA** [comment]
Unironically almost done. Just needs a few testmerges to see if there is anything bad/wrong with it that was missed in testing.

**Comment by YakumoChen** [comment]
Mild nitpick but is this supposed to be called 'Pubberstation' or 'Burgerstation' because its named the latter in game, but the former in your PR?

**Comment by YakumoChen** [comment]
Noticed solars access rooms lack air vents, so they can refill with air when engineers go in and out

Happy with more xeno spawns but they still need to be in places with vents they can crawl into (like solars, hint hint)

**Comment by YakumoChen** [comment]
Oh god there is a zauker tank
Please do not make a supply of roundstart instant-kill gas.

**Comment by Majkl-J** [comment]
Unreachable chargers in several locations, including medbay and bridge.
Example:
image

**Comment by Majkl-J** [comment]
+ Missing locker in chemistry, missing spare locker, wrong internal name (Burgerstation), lots of places that should have shutters (like chemistry) don't

**Comment by Majkl-J** [comment]
Another issue: Sleepers in medbay.
These are completely unnecessary and just dumb down medbay to "Input body, press 1-2 buttons, wait" and take any job from chemists.

**Comment by BurgerLUA** [comment]
> image Also this is stupid

This is not in the map and was constructed by a player.

**Comment by YakumoChen** [comment]
the map is missing a vacant commissary. I kinda think it should be where the gravgen is, because the gravgen is weirdly oversecure and not in the most friendly place (in a space island around the main hallway)

**Comment by Majkl-J** [comment]
image
What even is this

**Comment by Majkl-J** [comment]
Pharmacy windoor and door accesses are all wrong because ???

**Comment by Majkl-J** [comment]
image
Unreachable

File: _maps/map_files/burgerstation/burgerstation.dmm [hunk 1]
```diff
@@ -2684,8 +2685,8 @@
 /area/station/science)
 "biL" = (
 /obj/machinery/air_sensor/zauker_tank,
```
Comment (YakumoChen) [inline _maps/map_files/burgerstation/burgerstation.dmm]: change this to a plasma tank sensor too?

Honestly should probably just be co2 like standard on every other execution room but at least its not zauker

**Comment by Majkl-J** [comment]
image
Missing table

**Comment by Eastwardrope** [comment]
I just found a bug with brig atmos not working properly. Alot of the alarms such as in the brig proper and armoury aren't actually connected to the rooms properly and always state its 100 KPA I think they might be inside the wall properly instead of on the wall. Additionally when I set them to siphon it made the scrubbers red but didn't do anything until randomly all the air decided it shouldn't be.

**Comment by Eastwardrope** [comment]
Additionally in the vault the contents of one of the safes is just atop it instead of inside it. Its the safe with the 357

**Pull request merged**